README.md

```diff
@@ -101,7 +105,14 @@ This method also includes some overloads that do not require optional parameters
 ```java
 IncogniaAPI api = new IncogniaAPI("client-id", "client-secret", Region.BR);
 try {
-     TransactionAssessment assessment = api.registerLogin("installation-id", "account-id", "external-id");
+     RegisterLoginRequest registerLoginRequest =
+        RegisterLoginRequest.builder()
+          .installationId("installation id")
+          .accountId("account id")
+          .externalId("external id")
+          .evaluateTransaction(true) // can be omitted if you want this to be evaluated
```

Comment:
```suggestion
          .evaluateTransaction(true) // can be omitted as it uses true as the default value
```
What do you think?

I believe that we should also include a section explaining the usage of the eval parameter, as we do on the [Go lib](https://developer.incognia.com/docs/br/v6/apis/feedback/feedback-api#understanding-event-types).

Reply:
i'll add it

Reply:
done